cilium-cli/cli/bgp.go

```diff
@@ -52,7 +52,7 @@ func newCmdBgpPeers() *cobra.Command {
 	cmd.Flags().StringVar(&params.NodeName, "node", "", "Node from which BGP status will be fetched, omit to select all nodes")
 	cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 1*time.Minute, "Maximum time to wait for result, default 1 minute")
 	cmd.Flags().StringVarP(&params.Output, "output", "o", status.OutputSummary, "Output format. One of: json, summary")
-
+	cmd.Flags().BoolVarP(&params.ShowCaps, "caps", "c", false, "Show all BGP peer capabilities in detail")
```

Comment:
nit: The expanded flag could benefit from not shortening `capabilities`; this is much clearer if a user writes this in a script somewhere for instance:
```suggestion
	cmd.Flags().BoolVarP(&params.ShowCaps, "capabilities", "c", false, "Show all BGP peer capabilities in detail")
```

(More important for the user-facing cilium-cli, but applicable also to cilium-dbg command)